reward/reward_config_cache.go

```diff
@@ -59,21 +58,6 @@ func newRewardConfigCache(governanceHelper governanceHelper) *rewardConfigCache
 }
 
 func (rewardConfigCache *rewardConfigCache) get(blockNumber uint64) (*rewardConfig, error) {
-	var epoch uint64
-	pset, err := rewardConfigCache.governanceHelper.ParamsAt(blockNumber)
```

Comment:
@ian0371 
governanceHelper.ParamsAt() is also used at [line 92](https://github.com/klaytn/klaytn/pull/1637/files#diff-06d8f705f1e9a839705799e9d8a5b0b4845974798d40dd8b8c4b08ae62c16380R76) in this file. Should this be remained?

Reply:
@kjhman21 We're only removing epoch from cache. line 92 is to fetch the ratio at the given block number.

Reply:
@ian0371 So, you mean the data is stored based on the epoch in the DB?
reward config can be changed anytime, but DB stores data only when the block height is divisible by the epoch?